Backend/app/librarian/librarian_router.py

```diff
@@ -1,7 +1,10 @@
 # TODO: temp router development only, add access token requirements that check for librarian status.
 
-from fastapi import APIRouter
+from fastapi import APIRouter, Depends, HTTPException
+from fastapi.security import HTTPBearer
 
+from Backend.app.middlewares import verify_token
```

Comment:
You forgot to erase the `Backend.` that's automatically added by vscode in a number of places.